.babelrc

```diff
@@ -8,14 +8,15 @@
     "react"
   ],
   "env": {
-    "commonjs": {
+    "test": {
```

Comment:
We should have a `commonjs` env because that's one of our target `es` (ESM by default) vs. `lib` (CommonJS).

And it should **not** be instrumented. In this branch, running `yarn build-lib` and looking at `lib/**/*.js` now shows all our *library* code that we ship (e.g., https://unpkg.com/radium@0.25.0/lib/) is fully instrumented with a whole bunch of istanbul cruft.

Running `yarn build-lib` builds to `lib` now

Reply:
The `test` environment _could_ be added here, but `build-lib` needs to produce non-instrumented code. Which is why it was added in karma conf babel-loader before.

Reply:
If we added an additional env `test` along side the old unchanged `commonjs`, how would that affect the output of `build-lib` then?